POLICY.md

```diff
@@ -0,0 +1,65 @@
+    *	Use of new/malloc without matching delete/free (mem leak)
+5.	Issues about topics already handled in the documentation will be closed in a similar manner. Example:
+    *	can't flash with error espcomm failed
+6.	Issues must be provided with a minimalist sketch. Issues with an incomplete sketch, or a huge sketch, will be closed. Maximum effort must be put forth by the person opening the issue to reduce the relevant code that reproduces the issue, so that investigation can be taken up. MCVE is a must.
+7.	Issues with accompanied investigation that shows the root of the problem should be given priority
+8.	Duplicate issues will be closed with a reference to the original
+
+# Triaging
+1.	Any contributor of the project can participate in the triaging process, if he/she chooses to do so
+2.	An issue that needs to be closed, either due to not complying with this policy, or for other reasons, should be closed by a contributor
+3.	Issues that are accepted should be marked with appropriate labels, e.g.: component: xyz
+4.	If an issue is deemed to require specialized knowledge (e.g.: TLS, HTTP parser, SDK integration, etc), contributor(s) relevant to the affected code should be /cc’ed, as this can help grab attention by the right person
+5.	Severe issues should be assigned to the current milestone (i.e.: the next version to be released), or the milestone following. It is ok to push back issues depending on available resources.
+6.	Issues that could impact functionality for many users should be considered severe. 
+7.	Issues caused by the SDK or chip should not be marked severe, as there usually isn’t much to be done. Common sense should be applied when deciding. Such issues should be documented in a KID (Known Issues Document), possibly on the Wiki, for reference by users. Example:
+    *	ARP issue
+    * Extra channel change beacon announced by the SoftAP
+    *	Wakeup ROM bug in the ESP chip
+8.	Issues with feature requests should be discussed for viability/desirability. Example:
+    *	Support for new board. If the new board is not widely used, doesn’t have a manufacturer webpage, etc, then it isn’t desirable to support it. If the new board is essentially a duplicate of another, it isn’t desirable to duplicate the existing one.
+9.	Feature requests or changes that are meant to address a very specific/limited use case, especially if at the expense of increased code complexity, may be denied, or may be required to be redesigned, generalized, or simplified
+10.	Feature requests that are not accompanied by a PR:
+    *	could be closed immediately (denied)
+    *	could be closed after some predetermined period of time (left as candidate for somebody to pick up)
+    *	could be deemed interesting enough to work on, but without a specific project or target, and hence accumulated in a long-term feature request list. Such feature requests should in general not be targeted for a deadline or release.
```

Comment:
Based on the previous discussion i gather that the agreement is to go for option (2) here (close after some time unless someone expresses interest in working on an enhancement). Suggest removing options 1 and 3 and replacing 'could' with 'should'.